docs/content/controls/_checkbox.md

```diff
@@ -0,0 +1,43 @@
+#Checkbox
+A simple checkbox.
+
+*Returns* `string|empty` Returns the string `on` if checked or empty if unchecked
+
+##Example
+
+```php
+$this->add_control(
+	'show_title',
+	[
+		'label' => __( 'Show Title', 'your-plugin' ),
+		'type' => Controls_Manager::SWITCHER,
```

Comment:
Since this is a checkbox, maybe this should read `Controls_Manager::CHECKBOX` instead?